src/main/scala/org/broadinstitute/monster/encode/extraction/ExtractionPipeline.scala

```diff
@@ -99,14 +99,20 @@ object ExtractionPipeline {
       "library.accession"
     )
 
-    // don't need to use experiments RIGHT NOW apart from storing them, so we don't assign an output here
-    // TODO output experiments for future use of experiments to get files
-    extractLinkedEntities(
+    val experiments = extractLinkedEntities(
       EncodeEntity.Experiment,
       "experiment",
       replicates
     )
 
+    // don't need to use files apart from storing them, so we don't assign an output here
+    extractLinkedEntities(
+      EncodeEntity.File,
+      "@id",
+      experiments,
+      "dataset"
```

Comment:
Existing code, but: It's tough to know what these different `String` args correspond to when just looking at the call-sites. Can you update the call(s) to use a kwargs-style call?
```scala
extractLinkedEntities(
  encodeEntity = EncodeEntity.File,
  referenceField = "@id",
  prevData = experiments,
  entityFieldName = "dataset"
)
```

Reply:
(The kwargs would need to be updated to match the new arg names, if they get changed)